otherlibs/configurator/src/v1.ml

```diff
@@ -308,18 +309,34 @@ let read_dot_dune_configurator_file ~build_dir =
     in
     { ocamlc; vars }
 
+(* Needed for OCaml 4.03 compat, String.split_on_char was introduced in 4.04. *)
+let split_on_char s ~sep =
```

Comment:
Actually, this isn't unnecessary after all. `import.ml` already contains `split`. Sorry for the trouble.

Reply:
I've used `Configurator.V1.Flags.extract_blank_separated_words`.